int8 id

float32 width
float32 length
float32 height
float32 long_velocity
float32 lat_velocity
float32 long_accel

float32 long_position

float32 angle

float32 ext_angle_rate_left
float32 ext_angle_rate_right